src/main/resources/templates/fragments.html

```diff
@@ -8,16 +8,18 @@
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
 
    <link rel="stylesheet" th:href="@{/styles.css}">
-
 </head>
+<header>
+   <img th:src="@{/images/treasureTask-header.jpg}" alt="Header Image" width="200" height="150">
```

Comment:
We may want to move the image into the already existing header instead of making another header. I believe there should only be one header in the file.